### DISCUSSION TITLE
dotfiles does not setup the `~/.config` folder

### DISCUSSION THREAD

**Discussion by jonasbb** [community/community#14273] [Codespaces]
In the dotfile support documentation it states that:
> If none of these files are found, then any files or folders in your selected dotfiles repository starting with `.` are symlinked to the codespace's `~` or `$HOME` directory.
([Source](https://docs.github.com/en/codespaces/customizing-your-codespace/personalizing-codespaces-for-your-account#dotfiles))

This *should* also apply to the `~/.config` folder, but for me, the `.config` folder is not symlinked. Instead, it contains some subfolders which, I assume, are relevant for Codespaces / VS Code, like `configstore`, `procps`, `vscode-dev-containers`.
Many programs place their configuration there, and thus it is quite unfortunate, that this directory is not properly linked.

The directory seems to be cloned correctly from my dotfile repository:
```
codespace:/home/codespace> file ~/.config
/home/codespace/.config: directory

codespace:/home/codespace> file /workspaces/.codespaces/.persistedshare/dotfiles/.config
/workspaces/.codespaces/.persistedshare/dotfiles/.config: directory
```

In the `creation.log` I cannot find anything useful (from [troubleshooting](https://docs.github.com/en/codespaces/troubleshooting/troubleshooting-dotfiles-for-codespaces)):

```
$ # Clone & install dotfiles
dotfiles marker found


Stop (3 ms): Run in container: # Clone & install dotfiles
```

**Comment by zerkz** [comment]
I also struggled with this issue, trying to add files to `~/.config/fish/`
